hedera-node/hedera-app/src/main/java/com/hedera/node/app/roster/RosterService.java

```diff
@@ -68,7 +77,6 @@ public int migrationOrder() {
     @Override
     public void registerSchemas(@NonNull final SchemaRegistry registry) {
         requireNonNull(registry);
-        registry.register(new V0540RosterSchema());
-        registry.register(new V057RosterSchema(canAdopt, WritableRosterStore::new, ReadablePlatformStateStore::new));
```

Comment:
Don't we need a new schema for 0.58 ?

Reply:
I added the `restart()` implementation to the `054RosterSchema` [here](https://github.com/hashgraph/hedera-services/blob/16839-restart-hapi-tests/hedera-node/hedera-app/src/main/java/com/hedera/node/app/roster/schemas/V0540RosterSchema.java#L104) since it is the latest now.

Reply:
But that schema doesn't run if its version is 0.54 right ?

Reply:
Ah my bad. The `latestVersion` schema is 0.54 and the schema version is `0.54` . So we do run `restart` from latest schema available. Thanks